charts/ndm/crds/blockdevice.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: blockdevices.openebs.io
+spec:
+  additionalPrinterColumns:
```

Comment:
Can you add all the printer columns that are mentioned in [here](https://github.com/openebs/node-disk-manager/blob/43508b2424c81a994fb5ddd455cf0ce3351b0680/pkg/setup/build_crd.go#L28)

Reply:
done